config/config.go

```diff
@@ -258,10 +260,57 @@ type AuthenticationProviderConfig struct {
 	TokenValidityTimeout int
 }
 
+// ReadConfig read, parse and return a Configuration from the configuration file
+func ReadConfig(extraConfigFile string) (configuration Configuration) {
+	layers := make([]onion.Layer, 0)
+
+	layers = addFileLayers(layers,
+		"/opt/ercole/config.toml",
+		"/usr/share/ercole/config.toml",
+		"/etc/ercole.toml")
+
+	folderLayer, err := onion.NewFolderLayer("/etc/ercole.d/", "toml")
+	if err == nil {
+		layers = append(layers, folderLayer)
+	}
```

Comment:
Why do you choose the name FolderLayer instead of DirectoryLayer. It think DirectoryLayer is a better and precise name

Reply:
Yay!  I agree, done!